memory_core/memory_core_magma.py

```diff
@@ -292,17 +228,82 @@ def __init__(self,
         # Save as underlying circuit object
         self.underlying = FromMagma(circ)
 
+        # Enumerate input and output ports
+        # (clk and reset are assumed)
+        core_interface = get_interface(lt_dut)
+        cfgs = extract_top_config(lt_dut)
+        assert len(cfgs) > 0, "No configs?"
+
+        # We basically add in the configuration bus differently
+        # than the other ports...
+        skip_names = ["config_data_in",
+                      "config_write",
+                      "config_addr_in",
+                      "config_data_out",
+                      "config_read",
+                      "config_en",
+                      "clk_en"]
+
+        # Create a list of signals that will be able to be
+        # hardwired to a constant at runtime...
+        control_signals = []
+        # The rest of the signals to wire to the underlying representation...
+        other_signals = []
+
+
```

Comment:
Too many new lines. Did you run pycodestyle?

Reply:
fixed